src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
It's not a good practice to disable eslint rules unless absolutely necessary. Consider updating your component to have an associated label, or provide a valid reason for disabling the rule in a comment. 🧐